### PR TITLE
Support castanets in android.

### PR DESCRIPTION
**Pull request by suyambulingamrm**
This PR is to discuss the changes done for android support.

### CODE REVIEW

**Comment by hyowon** [comment]
Is Castanets on Android working with these commits?

**Comment by suyambulingamrm** [comment]
As I verified, 
I could see the downloaded resource are received on android. (Android receives Buffer Sync and sends back SyncAck msgs.)
And BufferSync msgs for TileData are also proper.
Still I could see only blank screen.
I am not sure if this is related to the posix code change or a different issue. I am checking on it.

Test method:
./chrome --disable-gpu 127.0.0.1
./out/myandroid/bin/chrome_public_apk launch --args='--type=renderer --server-address=IP'

**Comment by hyowon** [comment]
To summarize, loading and SW rasterizing are working but blank pages show up in some cases. Thank you for sharing.

**Comment by suyambulingamrm** [comment]
After enabling kEnableSurfaceSynchronization, I could see google page rendering properly
for linux(B)-android(R)

File: content/browser/android/content_startup_flags.cc [hunk 1]
```diff
@@ -26,6 +31,21 @@ void SetContentCommandLineFlags(bool single_process) {
...
+      service_manager::switches::kNoSandbox);
+  base::CommandLine::ForCurrentProcess()->AppendSwitch(switches::kNoZygote);
+  base::CommandLine::ForCurrentProcess()->AppendSwitch(
+      switches::kDisableGpuCompositing);
+  base::CommandLine::ForCurrentProcess()->AppendSwitchASCII(
+      switches::kNumRasterThreads, "4");
+  base::CommandLine::ForCurrentProcess()->AppendSwitchASCII(
+      switches::kLang, "en-US");
+  base::CommandLine::ForCurrentProcess()->AppendSwitchASCII(
+      switches::kEnableLogging, "stderr");
```
Comment (is46-kim) [inline content/browser/android/content_startup_flags.cc]: Need stderr for Android?

Comment (suyambulingamrm) [reply content/browser/android/content_startup_flags.cc]: Seems stderr is not required explicitly for android. I have removed the same in latest commit.

**Review by hyowon — APPROVED** [review]
https://github.com/Samsung/Castanets/issues/124#issuecomment-512675861
There is currently a performance issue, but this PR makes it possible for basic android running on Castanets. The performance task will be created and resolved separately.

**Review by is46-kim — APPROVED** [review]

**Pull request merged**